add example of targeting a Windows agent elastic stack (#2443)
buildkite/docs: pages/agent/v3/elastic_ci_aws.md

<%= image "buildkite-agent-token.png", size: "#{752/2}x#{424/2}", alt: "Reveal Agent Token" %>

> By default the stack uses a job queue of `default`, but you can specify any other queue name you like. 

Reviewer: This should fix the linting error:

```suggestion
By default the stack uses a job queue of `default`, but you can specify any other queue name you like.
```

A common example of setting a queue for a dedicated Windows agent can be achieved with the following in your `pipeline.yml` after you've set up your Windows stack:

```yaml
steps:
  - command: echo "hello from windows"
    agents:
      queue: "windows"
```

> See the [Buildkite Agent queue target docs](/docs/agent/v3/queues#targeting-a-queue) and [Buildkite Agent job queue docs](/docs/agent/v3/queues) for more info.

Reviewer: I think this reads a bit more smoothly with the links out:

```suggestion
For more information, see [Buildkite Agent job queues](/docs/agent/v3/queues), specifically [Targeting a queue](/docs/agent/v3/queues#targeting-a-queue).
```
